### DISCUSSION TITLE
Increase Prometheus Server Retention Period || Windows Server || Error

### DISCUSSION THREAD

**Discussion by Vivekdjango** [prometheus/prometheus#13378] [Q&A]
I got a request to increase retention period of already installed/running Prometheus server from default to 60 days.

Platform: Windows Server
Execution: NSSM
Prometheus Start Batch FIle:
nssm.exe install Prometheus D:\Observability\Prometheus\prometheus.exe "--web.listen-address=:9191"
nssm.exe set Prometheus Description "Prometheus metrics collection and monitoring service."
nssm.exe start Prometheus

What I have done so far:

a) nssm.exe restart Prometheus "--storage.tsdb.retention.time=60d".  (In days)
b) nssm.exe restart Prometheus "--storage.tsdb.retention.time=5184000"  ( In seconds)
c) Updated batch file and restarted the service
nssm.exe install Prometheus D:\Observability\Prometheus\prometheus.exe "--web.listen-address=:9191 --storage.tsdb.retention.time=60d"
nssm.exe set Prometheus Description "Prometheus metrics collection and monitoring service."
nssm.exe start Prometheus

But no positive effect.

Please help me to fix the issue as early as possible.

**Comment by markusleichter** [comment]
"nssm edit Prometheus" opens a GUI where you can add parameters
